chevah/compat
# [#3830] Move chevah.emprical to chevah.compat.testing

Scope
=====

The fact that in previous version chevah-compat package was installing both chevah.empirical and chevah.compat namespaces was creating a conflict with the existing chevah.empirical package.

Changes
=======

Moved all empirical to chevah.compat.testing and reorganized the chevah/compat/testing.py 

How to try and test the changes
===============================

reviewers: @alibotean 

check that changes make sense.

This will need to be tested as part of the work for 3828
